src/base/static/components/organisms/main-map.tsx

```diff
@@ -507,6 +507,7 @@ class MainMap extends React.Component<Props, State> {
               <Popup
                 latitude={this.state.popupLatitude}
                 longitude={this.state.popupLongitude}
+                closeOnClick={false}
```

Comment:
@goldpbear By default, react-map-gl closes the pop-up when it is clicked, which means that links inside of a popup aren't clickable because the popup will close and the link won't be opened. Since Brian wants to add links inside of his pop-ups, I decided to disable this `closeOnClick`. If someone wants to close a Popup, they'll have to hit the "close" button in the top right corner of the popup.

Hopefully this doesn't cause any issues.

Reply:
Sounds good-- I don't think this would cause any issues 👍